setup.sh

```diff
@@ -238,17 +249,31 @@ installDocker() {
   do
     echo
     writeBold "[❓] Do you want to install Docker now? (y/N)"
+    if [ $EUID != 0 ]; then
+      write "    Installing Docker needs root privileges."
+      write "    Since you are not running this script with root privileges"
+      write "    you are likely to be prompted for your sudo password."
+    fi
     echo -n "> "
     read installDocker trash
-    echo
     case $installDocker in
       [yY])
         if [ -z $DL_BIN ]; then
           failDL
         else
           writeBold "[ℹ] Installing Docker..."
-          $DL_BIN $DL_OPTS https://get.docker.com/ | sh
-          writeBold "$GREEN" "[✔] Docker successfully installed."
+          echo
+          sudo $DL_BIN $DL_OPTS https://get.docker.com/ | sudo sh
```

Comment:
I would display all sudo commands to the user.

Reply:
Ok, will do it in https://github.com/kuzzleio/kuzzle-build/pull/21, which actually handles the conditional `sudo` in a better way